tests/e2e/key/suite.go

```diff
@@ -117,4 +117,33 @@ var _ = ginkgo.Describe("[Key]", func() {
 		_, err = commands.CreateKey(keyName)
 		gomega.Expect(err).Should(gomega.HaveOccurred())
 	})
+
+	ginkgo.It("can list a created key", func() {
+		regex := `(\+-+\+\n)\|\s+KEY\sNAME\s+\|\n(\+-+\+\n)((\|\s+\w+\s+\|\n)(\+-+\+(\n|$)))+`
+
+		// Create a key
+		output, err := commands.CreateKey(keyName)
+		if err != nil {
+			fmt.Println(output)
+			utils.PrintStdErr(err)
+		}
+		gomega.Expect(err).Should(gomega.BeNil())
+
+		// Call list cmd
+		output, err = commands.ListKeys()
+		if err != nil {
+			fmt.Println(output)
+			utils.PrintStdErr(err)
+		}
+		gomega.Expect(err).Should(gomega.BeNil())
+
+		// The matcher for this test is a little weird. Instead of matching an exact
+		// string, we check that it matches a regex and contains created keyName. This
+		// is to facilitate running the test locally. If you have other keys in your
+		// key directory, they will be printed as well. It's impossible to check the
+		// list output for exact equality without removing pre-existing user keys.
+		// Hence, the matcher here.
```

Comment:
Note: if you were able to have the keys created in a tmp test dir, like we do with unit tests, it _would_ be possible. However, this being e2e tests, I understand that this probably amounts to unnecessary overhead. But worth having if possible.